sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/resolution/JoinResolver.kt

```diff
@@ -54,7 +54,7 @@ internal fun Resolver.resolve(
     } else {
       localResponse = resolve(table, recursiveCommonTable)
     }
-    errors.addAll(JoinValidator(this, localResponse, response + scopedValues.flatMap { it })
+    errors.addAll(JoinValidator(this, localResponse, scopedValues.plus<List<Result>>(response))
```

Comment:
i needed to specify the type cause the compiler thought i was trying to plus `List<Any>` for some reason